tests/kernel/uninstall_incident.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Disable kernel incident repositories and force package downgrade.
+#          Used mainly for livepatch uninstallation testing.
+# Maintainer: Martin Doucha <mdoucha@suse.cz>
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $repo = get_required_var('INCIDENT_REPO');
```

Comment:
It will be scheduled on on Incidents ?

Reply:
Yes, we run livepatch tests only in the single incidents group.